aiodocker/containers.py

```diff
@@ -190,6 +190,16 @@ def __init__(self, docker, **kwargs):
         await response.release()
         return
 
+    async def restart(self, **kwargs):
```

Comment:
I think we don't need kwargs. 

We can specify each arg: https://docs.docker.com/engine/api/v1.30/#operation/ContainerStop